pyomo/core/tests/unit/test_set.py

```diff
@@ -64,6 +64,12 @@
     Objective,
 )
 
+try:
+    import pandas as pd
+    pandas_available = True
+except:
+    pandas_available = False
```

Comment:
FYI: This is fine and does not need to be changed (as it is in a test file).  But for future reference, you can also:
```python
from pyomo.common.dependencies import pandas as pd, pandas_available
```
This leverages a deferred import mechanism so that the import is not attempted until either you cast `pandas_available` to a `bool`, or you use an attribute from `pandas`/`pd`.  As I mentioned, the deferred import is not as useful here, as you are guaranteed to need to check the availability, but in core code, this avoids the pandas import (which is very slow) unless you *actually* are going to need pandas.

Reply:
Neat! Good to know, thanks.